include/dpp/utility.h

```diff
@@ -420,6 +420,26 @@ namespace dpp {
 		 */
 		uint32_t DPP_EXPORT cmyk(int c, int m, int y, int k);
 
+		/**
+		 * @brief Convert doubles to HSL for sending in embeds
+		 *
+		 * @param h hue value, between 0 and 1 inclusive
+		 * @param s saturation value in procents, between 0 and 1 inclusive
```

Comment:
"procents"? Do you mean percent? Punctuation should also be here, with a capital letter and a full stop.